src/components/Auth/Login/Login.react.js

```diff
@@ -106,6 +120,7 @@ class Login extends Component {
           }
         })
         .catch(error => {
+          console.log(error);
```

Comment:
Is this needed ?

Reply:
oh.. nice, you just gave me an idea. I'll make some changes and resubmit the PR.

Reply:
haha cool